tests/accessibility/getAccessibilityTree.test.ts

```diff
@@ -0,0 +1,126 @@
+import type { ElementHandle } from 'pleasantest';
+import {
+  experimentalGetAccessibilityTree as getAccessibilityTree,
+  withBrowser,
+} from 'pleasantest';
+
+test(
+  'basic use cases',
+  withBrowser(async ({ utils, page }) => {
+    await utils.injectHTML(`
+      <main>
+        <button>
+          <span aria-hidden="true">+</span>
+          <span>Add to cart</span>
+        </button>
+        <h1>hiiii</h1>
+        <div role="button" tabindex="-1">foo &gt bar</div>
```

Comment:
TIL you don't need to end char codes with `;`

Reply:
Oops maybe the ever-forgiving HTML parser was just being nice to me 😂